src/Entity/Talk.php

```diff
@@ -101,6 +101,11 @@ public function removeSubmit(Submit $submit): self
         return $this;
     }
 
+    public function getAuthors()
+    {
+        return $this->getSubmits();
```

Comment:
The wording seems odd Author VS Submits, and anyway, I don't think this method is useful anyway :)